packages/web-components/src/menu-item/README.md

```diff
@@ -1,5 +1,3 @@
 # fast-menu-item
```

Comment:
Should we also rename these to "fluent"?

Reply:
Yes we should, thanks Rob. `fluent-menu` and `fluent-menu-item` are the only components with README so i created another work item too add READMEs to the rest.

https://github.com/microsoft/fluentui/issues/17318

Reply:
My other question is whether we should update the links so they point over to our fast specs, as opposed to removing the links.

Reply:
Yeah I chatted with @chrisdholt about that. We opted for removing them for now and when I go back and add the rest of the READMEs I'll add the proper links back.